src/main/java/io/github/bookcrawler/core/impl/publio/PublioBooksExtractor.java

```diff
@@ -0,0 +1,28 @@
+package io.github.bookcrawler.core.impl.publio;
+
+import io.github.bookcrawler.core.BookExtractor;
+import io.github.bookcrawler.core.SourceScrapper;
+import io.github.bookcrawler.core.impl.SourceScrappingResult;
+import io.github.bookcrawler.entities.BookInfo;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static io.github.bookcrawler.core.BookStore.PUBLIO;
+
+public class PublioBooksExtractor implements BookExtractor {
+    private SourceScrapper sourceScrapper;
+
+    public PublioBooksExtractor(SourceScrapper sourceScrapper) {
+        this.sourceScrapper = sourceScrapper;
+    }
+
+    @Override
+    public List<BookInfo> extract() {
+        return PUBLIO.crawler().crawl(PUBLIO.startUrl(), sourceScrapper).parallelStream()
```

Comment:
Just a hint here, but, what if you had one BookExtractor and `extract` would take as a parameter an enum? Because I have a feeling that the only thing that differs is PUBLIO or EMPIK or whatever else?